### PR TITLE
fix(security) remove session id parameter

### PR DESCRIPTION
**Pull request by loiclau**
## Description

The session id (sid) is set out in plain text on many pages.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [x] 2.8.x
- [x] 18.10.x
- [x] 19.04.x
- [x] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>
jira

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: www/include/monitoring/external_cmd/popup/popup.php [hunk 1]
```diff
@@ -60,7 +60,7 @@
 }
 $sid = session_id();
 if (isset($sid)) {
-    $res = $pearDB->prepare("SELECT * FROM session WHERE session_id = :sid");
+    $res = $pearDB->prepare("SELECT * FROM `session` WHERE `session_id` = :sid");
     $res->bindValue(':sid', $sid, PDO::PARAM_STR);
```
Comment (sc979) [inline www/include/monitoring/external_cmd/popup/popup.php]:
```suggestion
    $res->bindValue(':sid', session_id(), PDO::PARAM_STR);
```

File: www/include/monitoring/external_cmd/popup/popup.php [hunk 2]
```diff
@@ -60,8 +60,8 @@
 }
 $sid = session_id();
```
Comment (sc979) [inline www/include/monitoring/external_cmd/popup/popup.php]:
```suggestion
```

File: www/include/monitoring/external_cmd/popup/popup.php [hunk 3]
```diff
@@ -58,10 +58,9 @@
 ) {
     exit();
 }
-$sid = session_id();
 if (isset($sid)) {
```
Comment (sc979) [inline www/include/monitoring/external_cmd/popup/popup.php]:
```suggestion
if (isset(session_id())) {
```

File: www/include/common/javascript/ajaxMonitoring.js [hunk 4]
```diff
@@ -523,33 +523,15 @@ var _p = 20201;
...
-		_header.appendChild(_divdebug);
-		viewDebugInfo('--INIT--');
-	}*/
-	goM(_time_reload,_sid,_o);
+	goM(_time_reload,_o);
 }
 
-function goM(_time_reload,_sid,_o){
+function goM(_time_reload,_o){
 	// ici je recupere les couples host_name/service affichées sur ma page
```
Comment (bdauria) [inline www/include/common/javascript/ajaxMonitoring.js]: I don't understand this :)

**Pull request merged**